package.json

```diff
@@ -20,6 +20,7 @@
     "build:css": "csso src/autocomplete.css -o dist/accessible-autocomplete.min.css",
     "build:js": "cross-env NODE_ENV=production webpack --progress",
     "build": "run-s \"build:js\" \"build:css\"",
+    "postbuild": "node scripts/check-staged.mjs",
```

Comment:
Do we really want this to run whenever we run `npm run build` locally? I'm not sure I follow the thinking here.

Reply:
Yeah think so, for local development you'll run `npm run dev` instead

From our contributor PRs you'll see everyone misses it currently and follows up with a commit

Output will also stay silent unless `dist/` has changed

Reply:
> From our contributor PRs you'll see everyone misses it currently and follows up with a commit

Won't that still happen, if contributors are using `npm run dev` instead of `npm run build`?

Where are we expecting `npm run build` to be run where we want this to happen? Are there any places in CI where we're not adding `--ignore-scripts`?

I won't block this PR on it, but unless there's a clear reason for this change I'd be tempted to remove this change and have the 'Check for unstaged changes' job call `node scripts/check-staged.mjs` directly.

Reply:
>Won't that still happen, if contributors are using `npm run dev` instead of `npm run build`?

Nah we use [`webpack-dev-server`](https://webpack.js.org/configuration/dev-server/) so output isn't written to disk for `npm run dev`

We use `npm run build` for the "**To build the project for distribution**" docs + commit hook

I'm happy either way so I'll make your changes, not a problem